app/containers/HomeContainer/index.js

```diff
@@ -6,16 +6,19 @@ import { compose } from 'redux';
 import get from 'lodash/get';
 import debounce from 'lodash/debounce';
 import isEmpty from 'lodash/isEmpty';
-import { Card, Skeleton, Input } from 'antd';
 import styled from 'styled-components';
 import { injectIntl } from 'react-intl';
-import { useHistory } from 'react-router-dom';
-import T from '@components/T';
-import Clickable from '@components/Clickable';
 import { injectSaga } from 'redux-injectors';
+import { Card, Skeleton, Input } from 'antd';
+import T from '@components/T';
+import If from '@components/If';
+import For from '@app/components/For';
+import * as colors from '@app/themes/colors';
```

Comment:
```
import colors from '@app/themes/colors';
```
 should work. Let me know if you're having issues with it

Reply:
Done